README.md

```diff
@@ -1,14 +1,33 @@
-# Compute@Edge default template for Rust
+# Compute@Edge default starter kit for Rust
 
-Default package template used by the [Fastly CLI][cli] for Rust-based Compute@Edge projects.
+Get to know the Fastly Compute@Edge environment with a basic starter that demonstrates routing, simple synthetic responses, and overriding caching rules.
 
-## Usage
-- Install the [Fastly CLI][latest] to your `$PATH`
-- Run `fastly compute init`
+**For more details about this and other starter kits for Compute@Edge, see the [Fastly developer hub](https://developer.fastly.com/solutions/starters)**
 
-[cli]: https://github.com/fastly/cli
-[latest]: https://github.com/fastly/cli/releases/latest
+## Features
 
-## Security issues
+* Add headers to an inbound request
+* Allow only requests with particular HTTP methods
+* Match request URL path and methods for routing
+* Build synthetic responses at the edge
+* Send requests to a backend
+* Override caching policy response headers
 
-Please see our [SECURITY.md](SECURITY.md) for guidance on reporting security-related issues.
+## Understanding the code
+
+This starter is intentionally lightweight, and requires no dependencies aside from the [`fastly`](https://docs.rs/fastly) crate. It will help you understand the basics of processing requests at the edge using Fastly. This starter includes implementations of common patterns explained in our [using Compute@Edge](/learning/compute/using/) and [VCL migration](/learning/compute/migrate) guides.
+
+The path `/backend` will attempt to send a request to a backend called "backend_name".  If the service you have installed this starter on doesn't have a backend defined, use the [`fastly backend create`](/reference/cli/backend/create/) command to create one. Modify the following lines of the starter to use the name of the backend you created:
+
+```rust
+/// The name of a backend server associated with this service.
+///
+/// This should be changed to match the name of your own backend. See the the `Hosts` section of
+/// the Fastly WASM service UI for more information.
+const BACKEND_NAME: &str = "backend_name";
+
+/// The name of a second backend associated with this service.
+const OTHER_BACKEND_NAME: &str = "other_backend_name";
+```
+
+The starter uses two backends, so if you want to, go ahead and create two backends using the CLI and then modify both names here. You should now have a Fastly service running on Compute@Edge that can talk to your backends, and generate synthetic responses at the edge.
```

Comment:
I'd consider add an example CLI command to add a backend below this. Such as:
```suggestion
The starter uses two backends, so if you want to, go ahead and create two backends using the CLI and then modify both names here. You should now have a Fastly service running on Compute@Edge that can talk to your backends, and generate synthetic responses at the edge.

You can create a backend using the CLI:
```term
$ fastly backed create --name example --address www.example.com
```
```

Reply:
Better to link to canonical sources where they exist - and we actually already have a link to the relevant CLI command page in a preceding para.